### PR TITLE
Added Videos tab in Group Navigation

### LINKED ISSUE #2568: Add Videos tab in Group Navigation
**Describe the bug**
In Customize Group Navigation, I can’t reorder the Video tab to be between the Photos and Albums tabs - ‘Videos’ doesn’t show up in there (as a tab that can be moved), so it’s sitting oddly at the end of the whole menu right now.

**To Reproduce**
1. Go to Group Navigation
2. Check different menu items and they are draggable and  
3. We can set order of any menu

**Expected behavior**
Video menu on Group Navigation should be adjustable

### PR DESCRIPTION
**Pull request by krunal-buddyboss**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2568 

### How to test the changes in this Pull Request:

1.Go to Group Navigation
2.Check different menu items and they are draggable and
3.We can set order of any menu

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: src/bp-groups/classes/class-bp-groups-component.php [hunk 1]
```diff
@@ -491,7 +491,7 @@ public function setup_canonical_stack() {
 			$default_tab = bp_nouveau_get_temporary_setting( $customizer_option, bp_nouveau_get_appearance_settings( $customizer_option ) );
 		}
 
-		if ( 'photos' === $default_tab || 'albums' === $default_tab || 'documents' === $default_tab ) {
+		if ( 'photos' === $default_tab || 'albums' === $default_tab || 'documents' === $default_tab || 'videos' === $default_tab) {
```
Comment (KartikSuthar) [inline src/bp-groups/classes/class-bp-groups-component.php:494]:
```suggestion
		if ( 'photos' === $default_tab || 'albums' === $default_tab || 'documents' === $default_tab || 'videos' === $default_tab ) {
```

**Review by KartikSuthar — CHANGES_REQUESTED** [review]

**Review by chetansatasiya — DISMISSED** [review]
LGTM to except @KartikSuthar suggested changes

**Review by KartikSuthar — APPROVED** [review]
@krunal-buddyboss Code looks good to me. Merged to dev and pass to QA.

**Comment by sumitbuddyboss** [comment]
Hello @krunal-buddyboss 

Fixed >
Video tab added in group navigation and now we can reorder the menu as well for proof please check the below video.

Video:
https://www.loom.com/share/aafcb6486d67414895fdb678bdb01ce4

@chetansatasiya @sourabhmatolia 
Ready for release 🚀 

Thank you

**Comment by sourabhmatolia** [comment]
@sumitbuddyboss 

This is not working fine.

**Cases not handled properly:**
- IF customer re-ordered all tabs in master, now we add video tab in dev, then the tab order is wrong
- Courses, Sub Groups, Zoom all tabs show in Customizer but not in Frontend
- Newly added video tab should show after photos default but if the customer re-ordered group tabs before and now we add video tab then we should add a tab in the last.
- Need to cross-check all such cases for profile also.
- Check same for vertical layout also.

**Comment by sumitbuddyboss** [comment]
> @sumitbuddyboss
> 
> This is not working fine.
> 
> **Cases not handled properly:**
> 
> * IF customer re-ordered all tabs in master, now we add video tab in dev, then the tab order is wrong
> * Courses, Sub Groups, Zoom all tabs show in Customizer but not in Frontend
> * Newly added video tab should show after photos default but if the customer re-ordered group tabs before and now we add video tab then we should add a tab in the last.
> * Need to cross-check all such cases for profile also.
> * Check same for vertical layout also.

Hello @sourabhmatolia 

I and @krunal-buddyboss check all the points and here is the summary.

- IF customer re-ordered all tabs in master, now we add video tab in dev, then the tab order is wrong - **User need to update the orders first, We have checked that default videos menu showing at frontend side after the Photos.**

- Courses, Sub Groups, Zoom all tabs show in Customizer but not in Frontend - **These modules need to disable from admin from respective settings then it will be automatically removed from Group Navigation. You can check at** https://www.loom.com/share/9921e36c2be74ae4b3bb797e95880529

- Newly added video tab should show after photos default but if the customer re-ordered group tabs before and now we add video tab then we should add a tab in the last. - **Newly added video tab should show after photos default but if the customer re-ordered group tabs before then it will be shown before Manage and Send Message  menu tab.**

- Need to cross-check all such cases for profile also. - **Checked with Sumit that on the profile it works fine as excepted.**

- Check same for vertical layout also - Yes for vertical layout also it looks good.


@krunal-buddyboss 
Krunal create new issue regarding points you raised

**Pull request merged**